protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -41,6 +41,15 @@ message Value {
     }
 }
 
+// some existing schemas have time field as string and hence we should be able to accept such
+// time field into the system. Parsing of such fields will happen at data store layer
+message TimeField {
```

Comment:
Is there not an easy way to standardize the time field in logs-ingest?

Reply:
@sjkaris we would have to parse the field at log-ingest layer (and that is not cheap). Given that SCS already does the parsing logic of such timefield's, I think log-ingest should do basic validation and forward it

Reply:
Note: we would do validation of string representation to filter out invalid formats like "multiple dots", "non-numeric", etc